autoload/ale/lsp_linter.vim

```diff
@@ -420,24 +420,25 @@ endfunction
 
 function! s:HandleLSPResponseToCustomRequests(conn_id, response) abort
     if has_key(a:response, 'id')
-    \&& has_key(s:custom_callbacks_map, a:response.id)
-        let l:Callback = remove(s:custom_callbacks_map, a:response.id)
-        call l:Callback(a:response)
+    \&& has_key(s:custom_handlers_map, a:response.id)
+        let l:Handler = remove(s:custom_handlers_map, a:response.id)
+        call l:Handler(a:response)
```

Comment:
I thought using `Handler` here instead of `Callback` may help to make the code a bit more readable, since there is another layer of callbacks above it that are already called `Callback`s.